docs/AppFramework.md

```diff
@@ -338,11 +338,11 @@ Here is a typical App framework configuration in a Custom Resource definition:
   * The cluster scope is only supported on CRs that manage cluster-wide app deployment.
   
     | CRD Type          | Scope support                          | App Framework support |
-    | :---------------- | :------------------------------------- | :-------------------- |
+-------------------| :---------------- | :------------------------------------- | :-------------------- |
```

Comment:
is this indentation correct? how will this look in preview mode?

Reply:
Unintended fix probably from the IDE. Removing in the next commit.